### ISSUE TITLE
[Bug]: Linux Install Script tries to access incorrect folder

### ISSUE THREAD

**Issue by R2DToo** [ThioJoe/YT-Spammer-Purge#402]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

Downloaded and ran the install script for Linux as directed by the documentation here: https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Linux-Installation-Instructions#automatic-installation
During the scripts execution I think it is trying to access the requirements.txt file but it looks in the incorrect folder. 
The script is looking for the 'YouTube-Spammer-Purge-2.12.1' folder but an ls shows that the folders actual name is 'YT-Spammer-Purge-2.12.1'.
Check the Relevant Log Output section to see more information. Thanks :)

### Release version

2.12.1

### Steps to reproduce

https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Linux-Installation-Instructions#automatic-installation

### What platform are you seeing this problem on?

Linux (can also be ChromeOS)

### Relevant log output

```shell
./install_full.sh: line 60: cd: YouTube-Spammer-Purge-2.12.1/: No such file or directory
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
Dependencies and Program installed!
Now follow these instructions to get a client_secrets.json file!
https://github.com/ThioJoe/YouTube-Spammer-Purge#instructions---obtaining-youtube-api-key

╭─braden@braden in ~/Documents/coding_folder/YouTube-Spammer-Purge took 21s
╰─λ ls
drwxr-xr-x    - braden 18 Jan 20:31  YT-Spammer-Purge-2.12.1
.rwxr-xr-x 2.3k braden 19 Jan 16:41  install_full.sh
```

**Comment by ThioJoe** [comment]
Yea looks like that was the issue, should be fixed now with 626c5a2713f5281e6aeba19f7e5fdaf4d38853e3

**Comment by R2DToo** [comment]
Wow your so fast! Thanks so much for your work, I'm excited to try this out.
